jfrog_platform_cloud/README.md

```diff
@@ -0,0 +1,30 @@
+# Agent Check: JFrog Platform Cloud
+
+## Overview
+
+[JFrog][1] is a universal hybrid, end-to-end DevOps platform. This integration allows any JFrog cloud customer to seamlessly stream logs from JFrog Artifactory into Datadog. This integration provides Datadog [log pipelines][2] which enrich and index logs to make them more searchable and treatable using Datadog [facets][3].
```

Comment:
Could you provide more insight on the value customers will get from this integration?

Reply:
Added.

Reply:
@danielmkn apologies for the confusion - To clarify, the overview should include: 

1. What does your product do (in 1-2 sentences)?
2. What will customers get from this integration, and why is it valuable to them?
3. What types of data will your integration send in or pull out, and what's the value of that data? ex: By sending logs to Datadog, you have visibility into xyz

Reply:
Done.